config/main.sample.yaml

```diff
@@ -4,3 +4,6 @@ logLevel: info
 validators:
 - name: test
   address: Gt6HqWBhdu4Sy1u8ASTbS1qf2Ac5gwdegwr8tWN8saMxPt5
+  # expected:
```

Comment:
I assume you want to have an actual working sample with the new format?

Reply:
this is just to say that these are optional values... should it be more specific ?